src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
Globally applying it to `.full` doesn't work, because that brings in the app list. I'll do a small sweep to try to find things that it should apply to.